Classification dataset cleaning

AlexeyVatolin opened the pull request:
<!-- If you are submitting a dataset or a model for the model registry please use the corresponding checklists below otherwise feel free to remove them. -->

Fixes [comment](https://github.com/embeddings-benchmark/mteb/issues/1049#issuecomment-2468424773) from #1049
### Scores comparison
As expected, scores slightly drop after my changes for most tasks.
**multilingual-e5-small**
|    | task_name                                 |   main_score_old |   main_score_new |   delta_percent |
|---:|:------------------------------------------|-----------------:|-----------------:|----------------:|
|  0 | AJGT                                      |         0.7455   |         0.727835 |           -2.37 |
|  2 | BengaliDocumentClassification             |         0.506934 |         0.496289 |           -2.1  |
|  4 | BengaliHateSpeechClassification           |         0.492376 |         0.497559 |            1.05 |
|  6 | BengaliSentimentAnalysis                  |         0.831888 |         0.805349 |           -3.19 |
|  8 | CSFDCZMovieReviewSentimentClassification  |         0.275586 |         0.277197 |            0.58 |
| 11 | CzechProductReviewSentimentClassification |         0.507227 |         0.504395 |           -0.56 |
| 12 | HotelReviewSentimentClassification        |         0.494775 |         0.492434 |           -0.47 |
| 14 | OnlineStoreReviewSentimentClassification  |         0.362256 |         0.311644 |          -13.97 |
| 16 | RestaurantReviewSentimentClassification   |         0.635986 |         0.631377 |           -0.72 |
| 18 | TweetEmotionClassification                |         0.511279 |         0.491574 |           -3.85 |
| 20 | TweetSarcasmClassification                |         0.609289 |         0.564206 |           -7.4  |

**paraphrase-multilingual-MiniLM-L12-v2**
|    | task_name                                 |   main_score_old |   main_score_new |   delta_percent |
|---:|:------------------------------------------|-----------------:|-----------------:|----------------:|
|  1 | AJGT                                      |         0.699278 |         0.693557 |           -0.82 |
|  3 | BengaliDocumentClassification             |         0.335498 |         0.300439 |          -10.45 |
|  5 | BengaliHateSpeechClassification           |         0.340113 |         0.323974 |           -4.75 |
|  7 | BengaliSentimentAnalysis                  |         0.606135 |         0.562747 |           -7.16 |
|  9 | CSFDCZMovieReviewSentimentClassification  |         0.268652 |         0.256982 |           -4.34 |
| 10 | CzechProductReviewSentimentClassification |         0.513428 |         0.493799 |           -3.82 |
| 13 | HotelReviewSentimentClassification        |         0.42041  |         0.421355 |            0.22 |
| 15 | OnlineStoreReviewSentimentClassification  |         0.32168  |         0.282877 |          -12.06 |
| 17 | RestaurantReviewSentimentClassification   |         0.604102 |         0.63818  |            5.64 |
| 19 | TweetEmotionClassification                |         0.385498 |         0.368708 |           -4.36 |
| 21 | TweetSarcasmClassification                |         0.540664 |         0.467872 |          -13.46 |
<!-- add additional description, question etc. related to the new dataset -->


### Code Quality
<!-- Please do not delete this -->
- [x] **Code Formatted**: Format the code using `make lint` to maintain consistent style.

### Documentation
<!-- Please do not delete this -->
- [ ] **Updated Documentation**: Add or update documentation to reflect the changes introduced in this PR.

### Testing
<!-- Please do not delete this -->
- [x] **New Tests Added**: Write tests to cover new functionality. Validate with `make test-with-coverage`.
- [x] **Tests Passed**: Run tests locally using `make test` or `make test-with-coverage` to ensure no existing functionality is broken.


### Adding datasets checklist
<!-- see also https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md -->

**Reason for dataset addition**: ... <!-- Add reason for adding dataset here. E.g. it covers task/language/domain previously not covered -->

- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [ ] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [ ] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [ ] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [ ] Run tests locally to make sure nothing is broken using `make test`.
- [ ] Run the formatter to format the code using `make lint`.


### Adding a model checklist
<!--
When adding a model to the model registry
see also https://github.com/embeddings-benchmark/mteb/blob/main/docs/reproducible_workflow.md
-->

 - [ ] I have filled out the ModelMeta object to the extent possible
 - [ ] I have ensured that my model can be loaded using
   - [ ] `mteb.get_model(model_name, revision)` and
   - [ ] `mteb.get_model_meta(model_name, revision)`
 - [ ] I have tested the implementation works on a representative set of tasks.

Samoed commented on mteb/abstasks/data_cleaning.py:
I think you should create `v2` versions of these tasks to not change existing results

KennethEnevoldsen replied:
Yeah, so to ensure backward compatibility of the benchmarks, we need to keep the original version and create a v2 of the task.

"STS22.v2" is a good example of this (though I probably wouldn't duplicate the metadata).

KennethEnevoldsen replied:
I would probably update the description of the task, though. Something like:

"This task version fixes discovered [quality issues](link to PR) by removing empty strings (XX%), duplicates (XX%) and leakage in the test set from the training set (XX%)."

If we downsample, I would add:

"We additionally downsample the dataset to N samples."

If we add a test set then I would reformulate to:

"This task version fixes discovered [quality issues](link to PR) by removing empty strings (XX%), duplicates (XX%), we additionally also introduce a test set from the original dataset consisting of N samples."

AlexeyVatolin replied:
Do you mean that I should create a new issue describing the changes for this pull request? I thought that issue #1049 was like epic where I would link all tasks related to data cleaning.

KennethEnevoldsen commented on mteb/tasks/Classification/ara/AJGT.py:
```diff
@@ -37,3 +40,14 @@ class AJGT(AbsTaskClassification):
     )
+
+    def dataset_transform(self):
+        self.dataset["test"] = self.dataset["train"]
```
Won't this the overwritten in the following (if not it should ensure 100&% leakage)
```suggestion
```

KennethEnevoldsen commented on mteb/tasks/Classification/ara/AJGT.py:
```diff
@@ -37,3 +40,14 @@ class AJGT(AbsTaskClassification):
 """,
     )
+
+    def dataset_transform(self):
```
@Samoed, to avoid the dataset transform every time, is it possible to just push the new version directly to HF (I seem to recall this was only added in v2)?

If not, we can keep the transform here from v1 and remove it in v2. WDYT?

Samoed replied:
Yes, it is possible. Datasets are uploading after dataset load where transformer is called

KennethEnevoldsen replied:
Ah so it doesn't include the transform?

Samoed replied:
No, transform in included in load_dataset

KennethEnevoldsen replied:
Check - so we would have to remove the dataset_transform on re-upload (that is perfectly fine)

KennethEnevoldsen commented on mteb/tasks/Classification/ara/AJGT.py:
```diff
@@ -16,7 +19,7 @@ class AJGT(AbsTaskClassification):
         category="s2s",
         modalities=["text"],
-        eval_splits=["train"],
+        eval_splits=["test"],
```
I would only say "test" here if there was actually a "test" set on the hub, otherwise it will probably cause some confusion.

KennethEnevoldsen reviewed (commented):
This looks good!

I suspect it might be ideal to move this to v2 due to the dataset upload (see below). @Samoed what do you think? 

Would it be relevant to remake into something like:

```
task = mteb.get_task(...)

from mteb._quality_control import deduplicate, remove_leakage
# the _ is to denote it isn't intended public functions

task = deduplicate(task)
task = remove_leakage(task)
task.metadata.description += NEW_DESC # see below
task.metadata.name += ".v2" # has to be smarter in case it is v2 it of course have to be v3
task.push_to_hub()
```

Once the PR is done we of course also need to add a test to ensure that new dataset additions doesn't have these problems - we can test this from the datasets statistics (computed for all in v2)

AlexeyVatolin commented:
I have refactored the code as you suggested. For each modified dataset, I created a new class with the .v2 suffix and uploaded the revised datasets to the mteb team space.

For datasets that did not originally have a test split - only a training set - I kept 2048 examples in the training set (from which 8 examples per class are sampled during classifier training), and used the remainder as the test set.

Below is a report summarizing the number of lines removed for each issue. The cleaning functions were applied in the order listed in the table:

- **create_test_split** - splitting the training set into train and test subsets
- **filter_controversial** - removing examples with identical text (after lowercasing and stripping leading spaces) but different labels
- **filter_empty** - removing empty texts
- **deduplicate** - removing duplicate texts
- **filter_leakage** - removing examples that appear in both the training and test sets

**AJGT**
| split | original size |
|:--------------------------|:---------------:|
| train                     |          1800 |

Cleaning report:
| filter                    | split          | removed |
|:--------------------------|:---------------|--------:|
| create_test_split         | train_to_test |     900 |

**OnlineStoreReviewSentimentClassification**

| split | original size |
|:--------------------------|:---------------:|
| train                     |          2415 |

Cleaning report:
| filter                    | split          | removed |
|:--------------------------|:---------------|--------:|
| create_test_split         | train_to_test |    1207 |
| filter_controversial      | train         |      79 |
| filter_controversial      | test          |      64 |
| deduplicate               | train         |      38 |
| deduplicate               | test          |      29 |
| filter_leakage            | test          |      40 |

**HotelReviewSentimentClassification**

| split | original size |
|:--------------------------|:---------------:|
| train                     |        105698 |

Cleaning report:
| filter                    | split          | removed |
|:--------------------------|:---------------|--------:|
| create_test_split         | train_to_test |    2048 |
| filter_controversial      | train         |     130 |
| filter_controversial      | test          |       1 |
| deduplicate               | train         |    1533 |
| deduplicate               | test          |       4 |
| filter_leakage            | test          |      40 |

**RestaurantReviewSentimentClassification**

| split | original size |
|:--------------------------|:---------------:|
| train                     |          8364 |

Cleaning report:
| filter                    | split          | removed |
|:--------------------------|:---------------|--------:|
| create_test_split         | train_to_test |    2048 |
| filter_controversial      | train         |       2 |
| deduplicate               | train         |      35 |
| deduplicate               | test          |       5 |
| filter_leakage            | test          |      23 |
AJGT is unchanged

**TweetEmotionClassification**

| split | original size |
|:--------------------------|:---------------:|
| train                     |         10065 |

Cleaning report:
| filter                    | split          | removed |
|:--------------------------|:---------------|--------:|
| create_test_split         | train_to_test |    2048 |
| filter_controversial      | train         |      15 |
| filter_controversial      | test          |       1 |
| filter_empty              | train         |       1 |
| deduplicate               | train         |      28 |
| deduplicate               | test          |       1 |
| filter_leakage            | test          |       7 |

**TweetSarcasmClassification**

| split | original size |
|:--------------------------|:---------------:|
| train                     |          8437 |
| test                      |          2110 |

Cleaning report:
| filter                    | split          | removed |
|:--------------------------|:---------------|--------:|
| filter_controversial      | train         |      96 |
| filter_controversial      | test          |      18 |
| deduplicate               | train         |     216 |
| deduplicate               | test          |      13 |
| filter_leakage            | test          |     118 |


**BengaliHateSpeechClassification**

| split | original size |
|:--------------------------|:---------------:|
| train                     |          3418 |

Cleaning report:
| filter                    | split          | removed |
|:--------------------------|:---------------|--------:|
| create_test_split         | train_to_test |    1709 |
| filter_controversial      | train         |     111 |
| filter_controversial      | test          |     111 |
| deduplicate               | train         |      31 |
| deduplicate               | test          |      28 |
| filter_leakage            | test          |      55 |

**BengaliSentimentAnalysis**

| split | original size |
|:--------------------------|:---------------:|
| train                     |         11807 |

Cleaning report:
| filter                    | split          | removed |
|:--------------------------|:---------------|--------:|
| create_test_split         | train_to_test |    2048 |
| filter_controversial      | test          |       2 |
| deduplicate               | train         |     829 |
| deduplicate               | test          |     112 |
| filter_leakage            | test          |     111 |

**BengaliDocumentClassification**

| split | original size |
|:--------------------------|:---------------:|
| train                     |        220574 |
| validation                |          4994 |
| test                      |         15012 |

Cleaning report:
| filter                    | split          | removed |
|:--------------------------|:---------------|--------:|
| filter_controversial      | train         |     332 |
| filter_controversial      | validation    |       2 |
| filter_controversial      | test          |      19 |
| deduplicate               | train         |     112 |
| filter_leakage            | test          |       5 |

AlexeyVatolin commented:
Here is a comparison of the results. The scores have changed slightly because I updated the train/test split.

**multilingual-e5-small**
| task_name                                 |   main_score_old |   main_score_new |   delta_percent |
|:------------------------------------------|-----------------:|-----------------:|----------------:|
| AJGT                                       |        0.745500 |        0.740667 |           -0.65 |
| BengaliDocumentClassification              |        0.506934 |        0.511182 |            0.84 |
| BengaliHateSpeechClassification            |        0.492376 |        0.496289 |            0.79 |
| BengaliSentimentAnalysis                   |        0.831888 |        0.815144 |           -2.01 |
| HotelReviewSentimentClassification         |        0.494775 |        0.495557 |            0.16 |
| OnlineStoreReviewSentimentClassification   |        0.362256 |        0.317225 |          -12.43 |
| RestaurantReviewSentimentClassification    |        0.635986 |        0.642426 |            1.01 |
| TweetEmotionClassification                 |        0.511279 |        0.477538 |           -6.60 |
| TweetSarcasmClassification                 |        0.609289 |        0.567364 |           -6.88 |

**paraphrase-multilingual-MiniLM-L12-v2**
| task_name                                 |   main_score_old |   main_score_new |   delta_percent |
|:------------------------------------------|-----------------:|-----------------:|----------------:|
| AJGT                                       |        0.699278 |        0.714000 |            2.11 |
| BengaliDocumentClassification              |        0.335498 |        0.334521 |           -0.29 |
| BengaliHateSpeechClassification            |        0.340113 |        0.331887 |           -2.42 |
| BengaliSentimentAnalysis                   |        0.606135 |        0.588922 |           -2.84 |
| HotelReviewSentimentClassification         |        0.420410 |        0.429406 |            2.14 |
| OnlineStoreReviewSentimentClassification   |        0.321680 |        0.291713 |           -9.32 |
| RestaurantReviewSentimentClassification    |        0.604102 |        0.630842 |            4.43 |
| TweetEmotionClassification                 |        0.385498 |        0.365081 |           -5.30 |
| TweetSarcasmClassification                 |        0.540664 |        0.485110 |          -10.28 |

KennethEnevoldsen commented on mteb/tasks/Classification/ben/BengaliDocumentClassification.py:
```diff
@@ -51,3 +52,49 @@ def dataset_transform(self) -> None:
+""",
+    )
+
+    def dataset_transform(self) -> None:
+        self.dataset = self.stratified_subsampling(
+            self.dataset, seed=self.seed, splits=["test"]
+        )
```
delete?

KennethEnevoldsen commented on mteb/tasks/Classification/ara/AJGT.py:
```diff
@@ -37,3 +38,38 @@ class AJGT(AbsTaskClassification):
+            "path": "mteb/ajgt",
+            "revision": "6c3a9c1aaf2d9fd5e41508c33eb250cd6804ce98",
+        },
+        description="Arabic Jordanian General Tweets (AJGT) Corpus consisted of 1,800 tweets annotated as positive and negative. Modern Standard Arabic (MSA) or Jordanian dialect.",
```
We need a description of the changes here as well, so that people know what the differences between the two are.

KennethEnevoldsen reviewed (commented):
Thanks for the detailed description!

I would probably deduplicate before creating the train-test split to prevent the leakage, but it comes out the same, I suppose.

>  I kept 2048 examples in the training set 

What if it has less than 2k? Seems like this is the case for AJGT?

Seems like AJGT didn't have any issues, any reason to fix it?

KennethEnevoldsen commented on mteb/tasks/Classification/ara/HotelReviewSentimentClassification.py:
```diff
@@ -43,3 +44,38 @@ def dataset_transform(self):
+            "path": "mteb/hotel_review_sentiment",
+            "revision": "9dd15890ce97da846d75dc329b70d3b8398c5542",
+        },
+        description="HARD is a dataset of Arabic hotel reviews collected from the Booking.com website.",
```
We need to know why it was updated.

KennethEnevoldsen replied:
(I commented a few other, but not all)

KennethEnevoldsen commented on mteb/tasks/Classification/ara/OnlineStoreReviewSentimentClassification.py:
```diff
@@ -33,3 +34,29 @@ def dataset_transform(self):
+            "path": "mteb/online_store_review_sentiment",
+            "revision": "11ba0645b00bb68f34e741956d07f19eb0526c92",
+        },
+        description="This dataset contains Arabic reviews of products from the SHEIN online store.",
```
same

KennethEnevoldsen commented on mteb/tasks/Classification/ara/RestaurantReviewSentimentClassification.py:
```diff
@@ -44,3 +45,38 @@ def dataset_transform(self):
+            "path": "mteb/restaurant_review_sentiment",
+            "revision": "04df217aa4274e0806e879ce8b76754d57544343",
+        },
+        description="Dataset of 8364 restaurant reviews from qaym.com in Arabic for sentiment analysis",
```
same - I suppose the numbers don't match as well

KennethEnevoldsen commented on mteb/tasks/Classification/ara/TweetEmotionClassification.py:
```diff
@@ -44,3 +45,38 @@ def dataset_transform(self):
+            "path": "mteb/tweet_emotion",
+            "revision": "0a5ed32f2f7ab21271de1323e6f3daaedf6c7874",
+        },
+        description="A dataset of 10,000 tweets that was created with the aim of covering the most frequently used emotion categories in Arabic tweets.",
```
same + numbers probably don't match

AlexeyVatolin replied:
It's funny that 10,000 is also an incorrect number, as there were originally 10,065 rows

KennethEnevoldsen reviewed (approved):
I think this essentially there. I would be great if it was clear for the description why we updated the task (I know that I have had issues with the current version changes that this wasn't clear). 

If we want to automate the fixes, the it might be possible to simply link to an issue/PR.

KennethEnevoldsen commented:
Sounds great!

Added a comment in the other PR on the text on the phrasing on the message, but otherwise I think we are good.

A thing that might be nice to do is also check for:
1) Empty string
2) Minimal length, e.g. like minimum 3 words (this one might be too hard, to generalize across dataset + languages)

(sorry for adding it now, if it is too much of a hassle feel free to ignore it)

AlexeyVatolin commented:
@KennethEnevoldsen, I've already added filtering for empty strings; you can see this in the results for the **TweetEmotionClassification** task. I'll think about how to add filtering based on the minimum number of words and will update here. It probably makes sense to have separate filters for languages with ideograms and for Arabic languages. Perhaps you have ideas on what minimum length conditions would be appropriate for each writing system?

Samoed commented on mteb/tasks/Classification/ara/AJGT.py:
```diff
@@ -37,3 +38,39 @@ class AJGT(AbsTaskClassification):
+
+
+class AJGTV2(AbsTaskClassification):
+    metadata = TaskMetadata(
```
Can you also add `adapted_from` for all new datasets?

Samoed replied:
@AlexeyVatolin Can you add this to your tasks?

KennethEnevoldsen commented:
> Perhaps you have ideas on what minimum length conditions would be appropriate for each writing system?

I think everything that is Latin script is probably reasonable to have at least 3 words (white space separated). At least for classification

For retrieval it can be a bit different as we often search in single words.

AlexeyVatolin commented:
@KennethEnevoldsen, I added the filter_short for all languages except "zho", "jpn", "tha" and "mya". I also changed the filter order slightly. The new order is: filter_empty, deduplicate, filter_short, create_test_split, filter_leakage, and filter_controversial.

Here is updated results:

## Original Sizes
| Task | Split | Original Size |
|------|:-----:|--------------:|
| AJGT | train | 1800 |
| HotelReviewSentimentClassification | train | 2048 |
| OnlineStoreReviewSentimentClassification | train | 2415 |
| RestaurantReviewSentimentClassification | train | 8364 |
| TweetEmotionClassification | train | 2048 |
| TweetSarcasmClassification | train | 8437 |
| TweetSarcasmClassification | test | 2110 |
| BengaliDocumentClassification | train | 220574 |
| BengaliDocumentClassification | validation | 4994 |
| BengaliDocumentClassification | test | 15012 |
| BengaliHateSpeechClassification | train | 3418 |
| BengaliSentimentAnalysis | train | 11807 |

## Cleaning Report
| Task | Filter | Split | Removed |
|------|--------|:-----:|--------:|
| AJGT | filter_short | train | 116 |
| AJGT | create_test_split | train_to_test | 842 |
| HotelReviewSentimentClassification | deduplicate | train | 4 |
| HotelReviewSentimentClassification | filter_short | train | 4 |
| HotelReviewSentimentClassification | create_test_split | train_to_test | 1020 |
| OnlineStoreReviewSentimentClassification | deduplicate | train | 223 |
| OnlineStoreReviewSentimentClassification | filter_short | train | 238 |
| OnlineStoreReviewSentimentClassification | create_test_split | train_to_test | 977 |
| RestaurantReviewSentimentClassification | deduplicate | train | 64 |
| RestaurantReviewSentimentClassification | filter_short | train | 144 |
| RestaurantReviewSentimentClassification | create_test_split | train_to_test | 2048 |
| TweetEmotionClassification | deduplicate | train | 1 |
| TweetEmotionClassification | filter_short | train | 30 |
| TweetEmotionClassification | create_test_split | train_to_test | 1008 |
| TweetSarcasmClassification | deduplicate | train | 259 |
| TweetSarcasmClassification | filter_short | train | 45 |
| TweetSarcasmClassification | deduplicate | test | 16 |
| TweetSarcasmClassification | filter_short | test | 15 |
| TweetSarcasmClassification | filter_leakage | test | 130 |
| BengaliDocumentClassification | filter_empty | train | 21 |
| BengaliDocumentClassification | deduplicate | train | 260 |
| BengaliDocumentClassification | filter_short | train | 3 |
| BengaliDocumentClassification | filter_empty | test | 1 |
| BengaliDocumentClassification | filter_short | test | 1 |
| BengaliDocumentClassification | filter_leakage | test | 23 |
| BengaliDocumentClassification | filter_controversial | train | 1 |
| BengaliDocumentClassification | filter_controversial | validation | 1 |
| BengaliHateSpeechClassification | deduplicate | train | 238 |
| BengaliHateSpeechClassification | filter_short | train | 14 |
| BengaliHateSpeechClassification | create_test_split | train_to_test | 1583 |
| BengaliSentimentAnalysis | deduplicate | train | 1053 |
| BengaliSentimentAnalysis | filter_short | train | 662 |
| BengaliSentimentAnalysis | create_test_split | train_to_test | 2048 |

## Results
**intfloat/multilingual-e5-small**
| task_name                                |   main_score_old |   main_score_new |   delta_percent |
|:-----------------------------------------|-----------------:|-----------------:|----------------:|
| AJGT                                     |         0.7455   |         0.754157 |            1.16 |
| BengaliDocumentClassification            |         0.506934 |         0.464876 |           -8.3  |
| BengaliHateSpeechClassification          |         0.492376 |         0.486812 |           -1.13 |
| BengaliSentimentAnalysis                 |         0.831888 |         0.793829 |           -4.58 |
| HotelReviewSentimentClassification       |         0.494775 |         0.488627 |           -1.24 |
| OnlineStoreReviewSentimentClassification |         0.362256 |         0.365609 |            0.93 |
| RestaurantReviewSentimentClassification  |         0.635986 |         0.629834 |           -0.97 |
| TweetEmotionClassification               |         0.511279 |         0.504365 |           -1.35 |
| TweetSarcasmClassification               |         0.609289 |         0.586147 |           -3.8  |

**sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2**
| task_name                                |   main_score_old |   main_score_new |   delta_percent |
|:-----------------------------------------|-----------------:|-----------------:|----------------:|
| AJGT                                     |         0.699278 |         0.705107 |            0.83 |
| BengaliDocumentClassification            |         0.335498 |         0.299153 |          -10.83 |
| BengaliHateSpeechClassification          |         0.340113 |         0.323596 |           -4.86 |
| BengaliSentimentAnalysis                 |         0.606135 |         0.549482 |           -9.35 |
| HotelReviewSentimentClassification       |         0.42041  |         0.43598  |            3.7  |
| OnlineStoreReviewSentimentClassification |         0.32168  |         0.290686 |           -9.64 |
| RestaurantReviewSentimentClassification  |         0.604102 |         0.622998 |            3.13 |
| TweetEmotionClassification               |         0.385498 |         0.399702 |            3.68 |
| TweetSarcasmClassification               |         0.540664 |         0.540534 |           -0.02 |

KennethEnevoldsen reviewed (approved):
I would like to make the PR link a [hyperlink](https://github.com/embeddings-benchmark/mteb/pull/2632) instead as we will likely include it in the docs.
